src/DependencyInjection/Factory/MenuAdminFactory.php

```diff
@@ -101,4 +103,11 @@ private function loadExtensions(array $config, ContainerBuilder $container, XmlF
             }
         }
     }
+    
+    private function loadBurgovKeyValueForm(ContainerBuilder $container)
+    {
+        $bundles = $container->getParameter('kernel.bundles');
+
+        $container->setParameter('cmf_sonata_phpcr_admin_integration.menu.use_burgov_keyvalue_form', isset($bundles['BurgovKeyValueFormBundle']) );
```

Comment:
this ignores the value set in the configuration. please use the logic as in the [SEO DI extension](https://github.com/symfony-cmf/seo-bundle/blob/5900c572b10f9df58689e898feb1970bd381ef43/src/DependencyInjection/CmfSeoExtension.php#L390-L397) to check for the bundle if the value is auto, and fail if set to `true` and burgov is missing